### ISSUE TITLE
Photoframe Authorization failed

### ISSUE THREAD

**Issue by Meron0th** [mrworf/photoframe#217]
Hello! I'm loving your photoframe application so much I'm building one for my family this christmas!

Unfortunately, upon the authorization step using the same .json file that has worked before, I now get the following error:

> Uh oh, something went wrong...
> Please go to [github](https://github.com/mrworf/photoframe/issues) and see if this is a known issue, if not, feel free to file a [new issue](https://github.com/mrworf/photoframe/issues/new) with the following information:
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.7/dist-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/root/photoframe/routes/baseroute.py", line 62, in __call__
    return self.handle(self.app, **kwargs)
  File "/root/photoframe/routes/oauthlink.py", line 44, in handle
    return self.redirect(self.servicemgr.oauthStart(kwargs['service']))
  File "/root/photoframe/modules/servicemanager.py", line 199, in oauthStart
    return svc.startOAuth()
  File "/root/photoframe/services/base.py", line 274, in startOAuth
    return self._OAUTH.initiate()
  File "/root/photoframe/modules/oauth.py", line 122, in initiate
    self.rid = self.getRedirectId()
  File "/root/photoframe/modules/oauth.py", line 118, in getRedirectId
    r = requests.get('%s/?register' % self.ridURI)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 70, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/api.py", line 56, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 488, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/sessions.py", line 609, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python2.7/dist-packages/requests/adapters.py", line 497, in send
    raise SSLError(e, request=request)
SSLError: ("bad handshake: Error([('SSL routines', 'tls_process_server_certificate', 'certificate verify failed')],)",)
```
>Thank you for your patience

[Logs](https://github.com/mrworf/photoframe/files/13640044/debug.txt) from one attempt right after the error occurs, if those help at all
The error seems to be SSL certification failing. Searching here on the forums, it seems like this could be on my end or it could be on the website's end?
Visiting https://photoframe.sensenet.nu/?register=234234 gives me a hash. I also tried verifying a brand new Google Cloud/API on my current photoframe (with a fresh SD card and different pi) which also failed.

Thanks so much for any help

**Comment by mrworf** [comment]
This typically happens when your local time isn't correct. But I see from the logs that that isn't the case. The more likely reason is that the libraries are out of date and due to the age of the distribution, it cannot download the newer libraries.

You could try cloning the SDcard that works (it probably got all the changes needed before support was dropped) and use that in the new frame.

I hope to get time to refresh the project for a newer distro, but I cannot guarantee when that will happen.

**Comment by Meron0th** [comment]
Thanks for the reply. It's very odd, I was able to verify the library when I was planning the project a month ago, so whatever broke broke recently.

In all my troubleshooting pi's and SD cards I erased my working photoframe without thinking about it so neither functions now, that was my mistake, but whatever solution I find for the new frame I'll use on my old frame.

It appears this is happening to a few others so I would guess it's the libraries being out of date. I'll keep eyes on the [other thread](https://github.com/mrworf/photoframe/issues/218) for a solution, but I'll close this one as they're duplicates. 

I'll switch back to this project once a solution is found or you get the time for an update. Even if I don't ultimately use your software, your project inspired me to make this gift so thanks for your work!
